scd/Makefile

```diff
@@ -65,12 +66,22 @@ ifeq ($(SANITIZERS),y)
     LOCAL_CFLAGS += -lasan -fsanitize=address -fsanitize=undefined -fsanitize-recover=address
 endif
 ifeq ($(TRUSTME_SCHSM), y)
-    # If requested, we build sc-hsm support into trustme
-    LOCAL_CFLAGS += -DENABLESCHSM
+    # If requested, we build sc-cardservice with sc-hsm support into the scd
+    LOCAL_CFLAGS += -DCARDSERVICE_SCHSM
+    SC_CARDSERVICE = y
+    SRC_FILES += sc-hsm-lib/sc-hsm-cardservice.c
+endif
+ifeq ($(BNSE), y)
+    # If requested, we build sc-cardservice with bn-se support into the scd
+    SC_CARDSERVICE = y
+    LOCAL_CFLAGS += -DCARDSERVICE_BNSE
+    SRC_FILES += sc-hsm-lib/bn-se-cardservice.c
+endif
+ifeq ($(SC_CARDSERVICE), y)
```

Comment:
Maybe add comment to document that this Flag should not be directly used, but only in combination with BNSE and TRUSTME_SCHSM

Reply:
done